BackofficeBundle/Resources/public/coffee/extendView/generateId.coffee

```diff
@@ -17,7 +17,7 @@ extendView['generateId'] = {
   generateId: ->
     if @checkInputId()
       contentTypeId = @recupInput($(".generate-id-source"))
```

Comment:
Is the recupInput function really needed ? At least, the name should be updated to use english words.